crates/evm/core/src/era_revm/db.rs

```diff
@@ -25,27 +25,30 @@ use zksync_basic_types::{
 use zksync_types::{
     api::{BlockIdVariant, Transaction, TransactionDetails},
     StorageKey, ACCOUNT_CODE_STORAGE_ADDRESS, L2_ETH_TOKEN_ADDRESS, NONCE_HOLDER_ADDRESS,
-    SYSTEM_CONTEXT_ADDRESS,
+    SYSTEM_CONTEXT_ADDRESS, SYSTEM_CONTEXT_CURRENT_L2_BLOCK_INFO_POSITION,
 };
 
 use zksync_utils::{address_to_h256, h256_to_u256, u256_to_h256};
 
 #[derive(Default)]
 pub struct RevmDatabaseForEra<DB> {
     pub db: Arc<Mutex<Box<DB>>>,
+    // pub env: Arc<Mutex<Env>>,
```

Comment:
commented code. 

Is it enough now to have only the current bloc?

Reply:
Yeah, the `env` isn't tracked properly by foundry anyway, e.g. for the block after setUp the changes aren't applied. I figured previously it was easier to simply rely on storage.